plugin.rb

```diff
@@ -346,6 +346,19 @@ class ::ListController
     end
   end
 
+  add_to_class(:search_topic_list_item_serializer, :assigned_to_user_id) do
+    id = object.custom_fields[TopicAssigner::ASSIGNED_TO_ID]
+    id && id.to_i rescue nil
```

Comment:
I don't think `to_i` ever raise an exception. `nil.to_i == 0`

Reply:
Looking at code comment:

https://github.com/discourse/discourse-assign/blob/c4aefd5917be8f5b897cfede19fd120deb3c8a6c/plugin.rb#L394-L395

I think it's for array (`[].to_i`) cases.

Reply:
I see, I guess it might be worth adding the same comment?

Reply:
Makes sense, done.

Reply:
I highly recommend never using `rescue nil` because it rescues all and any type of errors.

Reply:
I've seen you recommend this before (and correctly.) I wonder if we can lint for it?